config/initializers/devise.rb

```diff
@@ -0,0 +1,259 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # config.secret_key = 'b842f33d64d493321409cd2a2884231b83988373c534719d5360b34c7d4dbce016280bba17c058d28f1352f53cbe2322546b3a3241d159e48427c6be0e04f5e8'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.